### PR TITLE
remove table['name'] from error outputs

### LINKED ISSUE #80: Undefined index in error handler for configRows
stejne jako https://github.com/keboola/db-extractor-mysql/issues/86

### PR DESCRIPTION
**Pull request by pivnicek**
fixes #80

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/MSSQL.php [hunk 1]
```diff
@@ -199,8 +199,7 @@ public function export(array $table): array
         } catch (\Throwable $e) {
             $this->logger->info(
                 sprintf(
-                    "[%s]: BCP command failed: %s. Attempting export using pdo_sqlsrv.",
-                    $table['name'],
+                    "The BCP export failed: %s. Attempting export using pdo_sqlsrv.",
```
Comment (tomasfejfar) [inline src/Keboola/DbExtractor/Extractor/MSSQL.php]: shouldn't you add the missing $outputTable instead of removing the brackets?

Comment (tomasfejfar) [reply src/Keboola/DbExtractor/Extractor/MSSQL.php]: Or remove it everywhere else to keep it consistent.

File: src/Keboola/DbExtractor/Extractor/MSSQL.php [hunk 2]
```diff
@@ -170,8 +170,8 @@ public function export(array $table): array
                     $exportResult['lastFetchedRow'] = $this->state['lastFetchedRow'];
                 }
                 $this->logger->warning(sprintf(
-                    "Query returned empty result. Nothing was imported for table [%s]",
-                    $table['name']
+                    "Query returned empty result. Nothing was imported into the table [%s]",
```
Comment (tomasfejfar) [inline src/Keboola/DbExtractor/Extractor/MSSQL.php]: I think that keeping the format of `[tableName]: Message` would make it easier to read...

**Comment by pivnicek** [comment]
Thanks @tomasfejfar .  I agree with you so I've unified the message format to `[tablename]: message`

File: src/Keboola/DbExtractor/Extractor/MSSQL.php [hunk 3]
```diff
@@ -230,8 +230,8 @@ public function export(array $table): array
                         $exportResult['lastFetchedRow'] = $this->state['lastFetchedRow'];
                     }
                     $this->logger->warning(sprintf(
-                        "Query returned empty result. Nothing was imported for table [%s]",
-                        $table['name']
+                        "[%s]: Query returned empty result so nothing was imported",
+                        $table['outputTable']
```
Comment (tomasfejfar) [inline src/Keboola/DbExtractor/Extractor/MSSQL.php]: Pushed update to set this to $outputTable

**Pull request merged**